Tiltfile

```diff
@@ -1,25 +1,27 @@
 # -*- mode: Python -*-
 
 k8s_yaml([
-'config/crd/bases/velero.io_backups.yaml',
-'config/crd/bases/velero.io_backupstoragelocations.yaml', 
-'config/crd/bases/velero.io_deletebackuprequests.yaml',   
-'config/crd/bases/velero.io_downloadrequests.yaml',   
-'config/crd/bases/velero.io_podvolumebackups.yaml',  
-'config/crd/bases/velero.io_podvolumerestores.yaml',  
-'config/crd/bases/velero.io_resticrepositories.yaml',
-'config/crd/bases/velero.io_restores.yaml',    
-'config/crd/bases/velero.io_schedules.yaml', 
-'config/crd/bases/velero.io_serverstatusrequests.yaml',  
-'config/crd/bases/velero.io_volumesnapshotlocations.yaml', 
+    'config/crd/bases/velero.io_backups.yaml',
+    'config/crd/bases/velero.io_backupstoragelocations.yaml',
+    'config/crd/bases/velero.io_deletebackuprequests.yaml',
+    'config/crd/bases/velero.io_downloadrequests.yaml',
+    'config/crd/bases/velero.io_podvolumebackups.yaml',
+    'config/crd/bases/velero.io_podvolumerestores.yaml',
+    'config/crd/bases/velero.io_resticrepositories.yaml',
+    'config/crd/bases/velero.io_restores.yaml',
+    'config/crd/bases/velero.io_schedules.yaml',
+    'config/crd/bases/velero.io_serverstatusrequests.yaml',
+    'config/crd/bases/velero.io_volumesnapshotlocations.yaml',
 ])
 
 # default values
 settings = {
-   "default_registry": "",
-   "enable_restic": False,
-   "create_backup_locations": False, 
-   "setup-minio": False, 
+    "default_registry": "",
+    "enable_restic": False,
+    "enable_debug": False,
+    "debug_continue_on_start": True,  # Continue the velero process by default when in debug mode
+    "create_backup_locations": False,
+    "setup-minio": False,
```

Comment:
Does it make sense to standardize these so that `setup-minio` becomes `setup_minio`?